.github/workflows/auto_create_version_bump_pr.yml

```diff
@@ -76,6 +76,13 @@ jobs:
           reviewers: "DataDog/apm-dotnet"
           body: "${{steps.changes.outputs.release_notes}}"
 
+      - name: "Close previous milestone"
```

Comment:
I put it there as it's less critical than the others, easy to fix in case it fails. I'd rather leave it there even though that may be only because I've broken too many things lately :/ (I tested it locally though, not saying it will work as I've learned the hard way recently :p)